back/api/models.py

```diff
@@ -144,10 +145,21 @@ class Metadata(models.Model):
     """
     Describes one or more Products. Every metadata can have one or more contact persons
     """
+
+    class MetadataAccessibility(models.TextChoices):
```

Comment:
Uh... I did not know that you could define a class in a class...

Reply:
Yep in Java you can also do it. Django works a lot with that, all Serializers based on models have one